src/lib.rs

```diff
@@ -24,7 +25,7 @@ extern "C" {
     fn log(s: &str);
 }
 
-#[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Debug, Clone, Copy, PartialOrd, Ord, PartialEq, Eq, Serialize, Deserialize)]
```

Comment:
I wish it was more obvious which modules brought in which traits. I can infer in this case, but I won't always be able to do t hat.

Reply:
traits have to be `use`'d just like other symbols. `Serialize` and `Deserialize` are the only traits here that aren't part of the rust standard library, and you can see them being imported above:

```
use serde::{Deserialize, Serialize};
```

One option we have for making this clearer would be to declare a local style that means some references (like maybe deriving traits) must use the absolute path, which would make this line:

```
#[derive(Debug, Clone, Copy, PartialOrd, Ord, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
```